### PR TITLE
[Update] Remove Spacing and Add Private

### PR DESCRIPTION
**Pull request by suho**
In this pull request, I added private and removed/added spacing if needed.

### CODE REVIEW

**Review by NutanNiraula — APPROVED** [review]
LGTM

File: Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___PresenterSpec.swift [hunk 1]
```diff
@@ -15,11 +15,11 @@ final class ___VARIABLE_moduleName___PresenterSpec: QuickSpec {
 
     override func spec() { 
 
-        var presenter: ___VARIABLE_moduleName___Presenter!
-        var router: ___VARIABLE_moduleName___RouterInputMock!
-        var interactor: ___VARIABLE_moduleName___InteractorInputMock!
-        var view: ___VARIABLE_moduleName___ViewInputMock!
-        var output: ___VARIABLE_moduleName___OutputMock!
+        private var presenter: ___VARIABLE_moduleName___Presenter!
```
Comment (nmint8m) [inline Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___PresenterSpec.swift]: No need to do this.

**Review by nmint8m — CHANGES_REQUESTED** [review]
Please check again.

File: Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___RouterSpec.swift [hunk 2]
```diff
@@ -15,8 +15,8 @@ final class ___VARIABLE_moduleName___RouterSpec: QuickSpec {
 
     override func spec() { 
 
-        var router: ___VARIABLE_moduleName___Router!
-        var viewController: ___VARIABLE_moduleName___ViewController!
+        private var router: ___VARIABLE_moduleName___Router!
```
Comment (nmint8m) [inline Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___RouterSpec.swift]: No need to do this.

File: Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___ViewControllerSpec.swift [hunk 3]
```diff
@@ -15,8 +15,8 @@ final class ___VARIABLE_moduleName___ViewControllerSpec: QuickSpec {
 
     override func spec() { 
 
-        var viewController: ___VARIABLE_moduleName___ViewController!
-        var output: ___VARIABLE_moduleName___ViewOutputMock!
+        private var viewController: ___VARIABLE_moduleName___ViewController!
```
Comment (nmint8m) [inline Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___ViewControllerSpec.swift]: No need to do this.

**Review by nmint8m — COMMENTED** [review]
Please check again.

File: Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___PresenterSpec.swift (hunk 1 above)
Comment (suho) [reply Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___PresenterSpec.swift]: @nmint8m Please give me a reason that `private` is same with `internal` in this case

Comment (nmint8m) [reply Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___PresenterSpec.swift]: You define this inside function `spec()`, don't you?

**Review by nmint8m — APPROVED** [review]
LGTM!

**Pull request merged**